### ISSUE TITLE
Question about developCSM.m

### ISSUE THREAD

**Issue by lianghao0930** [Anwar-M/Acoustic-Beamforming#1]
Hi,
  Thank you for your nice code. 

  I have some questions. I am confused about why the coefficient, 0.5, is need to develop the CSM, i.e., (developCSM.m line 63)

  for F = 1:N_freqs
      CSM(:,:,F) = CSM(:,:,F) + **0.5***x_fft(freq_sels(F),:)'*x_fft(freq_sels(F),:);
  end

  Thank you so much!

**Comment by Anwar-M** [comment]
If I remember it is to have the correct power. You could omit it and then maybe later correct it during beamforming.

You can try to test it without the 0.5 and see if you get the power you want.

**Comment by lianghao0930** [comment]
Yep! After some experiments, I think the reason may be the different forms of Fourier transform, and we should let it obey Parseval's theorem. Thus, we need 0.5 to have the correct power. 

Thank you for your response!
